dynamic-loader/Makefile

```diff
@@ -0,0 +1,30 @@
+include ../Makefile.include
+
+SOURCES         = $(shell find *.c ! -name "sm1.c" -printf "%p ")
+OBJECTS         = $(SOURCES:.c=.o)
+SM1_HEADER      = $(shell python3 hardcode_elf.py)
+
+TARGET          = main.elf
+TARGET_NO_WRAP  = no_wrap_$(TARGET)
+
+LDFLAGS += -lsm-control -lloader -ltools
+
+all: $(TARGET)
+
+$(TARGET_NO_WRAP): $(SM1_HEADER) $(OBJECTS)
+	$(LD) $(LDFLAGS) --prepare-for-sm-text-section-wrapping -o $@ $^
```

Comment:
better to keep this example minimal without confidential loading. I understand the natural decision to base this on hello-world, but that one is unnecessarily complex w confidential loading, opening a separate issue for this #17